modules/landing-pages/documents/landing-page.php

```diff
@@ -1,26 +1,23 @@
 <?php
 namespace Elementor\Modules\LandingPages\Documents;
 
-use Elementor\Core\DocumentTypes\Page;
-use Elementor\Modules\LandingPages\Module;
+use Elementor\Core\DocumentTypes\PageBase;
 use Elementor\Modules\LandingPages\Module as Landing_Pages_Module;
-use Elementor\Modules\PageTemplates\Module as PageTemplatesModule;
+use Elementor\Modules\PageTemplates\Module as Page_Templates_Module;
 use Elementor\TemplateLibrary\Source_Local;
 
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly
 }
 
-class Landing_Page extends Page {
+class Landing_Page extends PageBase {
 
 	public static function get_properties() {
 		$properties = parent::get_properties();
 
 		$properties['support_kit'] = true;
 		$properties['show_in_library'] = true;
-
-		// This is so Landing Pages don't take over WordPress' native 'page' Post type.
-		unset( $properties['cpt'] );
+		$properties['cpt'] = [ 'landing-page' ];
```

Comment:
Rename it to `e-landing-page, create a constant and use it everywhere

Reply:
Done.